server/graphql/v2/object/PaymentMethod.js

```diff
@@ -26,6 +29,66 @@ export const PaymentMethod = new GraphQLObjectType({
           return paymentMethod.name;
         },
       },
+      service: {
+        type: GraphQLString,
+        resolve(paymentMethod) {
+          return paymentMethod.service;
+        },
+      },
+      type: {
+        type: GraphQLString,
+        resolve(paymentMethod) {
+          return paymentMethod.type;
+        },
+      },
+      balance: {
```

Comment:
Should be an `Amount` type.

Reply:
Currently the `balance` is an integer, and it's used externally in some payment method libs that rely on it being an int.

Making it an `Amount` type means it would be an object with subfields of `value` and `currency`. Should I go ahead and make this change and then also update the lib?

Reply:
Could you point me on the lib and function that would need an update?